README.md

```diff
@@ -37,6 +37,8 @@ In the project properties for **pdf-java-toolkit-samples**, select the **Maven**
 
 It will ask if you want to update the product configuration. Click **Yes**.
 
+The samples were developed and tested using [Mars](https://eclipse.org/mars/)
```

Comment:
I am on 4.5.0

@jrowden8, @datalogics-robl, and @datalogics-dans what versions are you guys on?